src/Docker.DotNet/Models/ContainerUpdateParameters.Generated.cs

```diff
@@ -98,12 +100,18 @@ public ContainerUpdateParameters(UpdateConfig UpdateConfig)
         [DataMember(Name = "Devices", EmitDefaultValue = false)]
         public IList<DeviceMapping> Devices { get; set; }
 
-        [DataMember(Name = "DiskQuota", EmitDefaultValue = false)]
-        public long DiskQuota { get; set; }
```

Comment:
This is a breaking change right? What replaced it?

Reply:
I am not sure. I looked at the docker API documentation and did not find anything called DiskQuota. 
(https://docs.docker.com/engine/api/v1.40/#operation/ContainerUpdate)
I also looked in the [repo](https://github.com/moby/moby/blob/b95fad8e51bd064be4f4e58a996924f343846c85/api/types/container/host_config.go) and did not find anything obvious.
The net is I do not even know where this field came from in the first place

Reply:
Super weird. This seems to have been removed in `v1.40`. But I cant see any notes on why. Last model it was in is: `https://docs.docker.com/engine/api/v1.39/#operation/ContainerUpdate`

Reply:
Yes, looks like it was remove moving to 1.40. I do not think it's a breaking change because likely the field will be ignored by docker. It might be perceived by users as regression though since the field is moot. At the end of the day I do not think we can do better than what docker api does.

Reply:
Agreed.